ui/src/app/pages/signed-in/nav-bar.tsx

```diff
@@ -89,48 +85,13 @@ export interface State {
 const barsTransformNotRotated = 'rotate(0deg)';
 const barsTransformRotated = 'rotate(90deg)';
 
-const cookieKey = 'status-alert-banner-dismissed';
```

Comment:
moved all of the status-alert-banner logic to... wait for it... status-alert-banner